qa/integration-tests/src/test/java/io/zeebe/broker/it/network/LargeMessageSizeTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it.network;
+
+import io.zeebe.broker.it.GrpcClientRule;
+import io.zeebe.broker.it.util.ZeebeAssertHelper;
+import io.zeebe.broker.test.EmbeddedBrokerRule;
+import io.zeebe.model.bpmn.Bpmn;
+import io.zeebe.model.bpmn.BpmnModelInstance;
+import io.zeebe.util.ByteValue;
+import java.util.Map;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.RuleChain;
+
+public class LargeMessageSizeTest {
+
+  private static final ByteValue MAX_MESSAGE_SIZE = ByteValue.ofMegabytes(4);
+  // only use half of the max message size because some commands produce two events
+  private static final ByteValue LARGE_SIZE = ByteValue.ofMegabytes(2);
```

Comment:
You could also test this just with KB would be probably more efficient

Reply:
True. But I really want to test the limits. Because of this large size, I discovered some configuration options ;)

Reply:
ok but now you found all parts right ?  :sweat_smile:

Reply:
I hope so :crossed_fingers: